website/docs/faqs/Accounts/transfer-account.md

```diff
@@ -19,3 +19,4 @@ When you make any account owner and email changes:
 
 - The new email address _must_ be verified through our email verification process. 
 - You can update any billing email address or [Notifications Settings](/docs/deploy/job-notifications) to reflect the new account owner changes, if applicable.
+ 
```

Comment:
hey @BrJan great flag! does this apply to all plans? I'm wondering if this is also something to mention here?

> When transferring account ownership, please ensure you [unlink](/faqs/Accounts/git-account-in-use) your GitHub account in dbt Cloud. This is because you can only have your Git account linked to one dbt Cloud user account.

Reply:
@mirnawong1 we just need to call this out for the Developer plan for security purposes.

That for sure would be a good not to add 👍.